static/styles/settings.css

```diff
@@ -718,22 +718,32 @@ input[type="checkbox"] {
     text-align: right;
 }
 
+#show-add-user-group-modal {
+    margin-bottom: 10px;
+}
+
+#add-user-group-form {
+    margin: 0;
+
+    /* This 14px is the border and padding of the input element */
+    #user_group_description {
+        width: calc(100% - 14px);
+    }
+}
```

Comment:
These are fine for now, but ideally we want to not require any custom CSS for these modals, so we should be thinking about how to achieve that by improving the commons styling and HTML choices.